### DISCUSSION TITLE
2FA bug

### DISCUSSION THREAD

**Discussion by paolini** [community/community#78255] [Other Feature Feedback, Questions, & Ideas]
I have 2FA correctly configured with the github app. This is my preferred authentication method. Nevertheless github is forcing me to complete the configuration of a TOTP application which I no longer have access to.

Now I can use github with my phone but cannot use it on my laptop. Seems there is no way to remove the TOTP authentication method and dismiss the request to complete configuration.

This situation is embarrassing. There is a real problem with how 2FA was implemented.

**Comment by VIPLightning** [comment]
Seems like forcing everyone to use 2FA wasn't well thought-out. But the moment people speak out against this anti-consumer/community behavior, those users that are using 2FA try to gaslight others while coming up with a strawman argument and insult you. Here's my philosophy, if I created my data on here and I have access to it after entering my credentials, I should be allowed to do what I want and instead of being locked out by some company because I refused to use their garbage and poorly implemented security. Unfortunately as 2FA was rolled in, I noticed more toxicity happening in the discussions. That's why I recommend switching to
* Codeberg 
* Sourceforge 
* bitbucket  

None of these git community providers force you into using 2FA. I have 2 days to use my account before github locks my account. And when that happens, that's when I stick my middle finger high at the Github staff and tell them to stick it. Github and Gitlab are no different from each other when it comes to shady behaviors but were built on top of git.

**Comment by virajbhutada** [comment]
It's understandable that dealing with 2FA configuration issues can be frustrating. To address this, try going to your [GitHub Security settings](https://github.com/settings/security) and navigate to the "Two-factor authentication" section. Look for options to configure or remove TOTP (Time-based One-Time Password). If you're unable to access the TOTP application, explore alternative methods provided, such as backup codes or other authentication options.

If the issue persists, consider reaching out to GitHub support for personalized assistance. Explain your situation, and they may be able to help you disable TOTP or provide further guidance. In the meantime, if you can access GitHub on your phone, consider using it as an alternative authentication method on your laptop until the issue is resolved. Your patience is appreciated, and your feedback is valuable for improving the platform.

**Reply by paolini** [reply]
I resolved by completing the configuration of the TOTP authentication method and then throwing it away. This, however, is a workaround of a process which is not well implemented. You should not be locked out if you actually have an authentication method available.
